src/components/Menu.vue

```diff
@@ -95,29 +94,51 @@
       })
     },
     methods: {
-      async closeMenu() {
-       await menuController.close();
-      },
       async setEComStore(event: CustomEvent) {
         if(this.eComStore.productStoreId !== event.detail.value) {
           await this.store.dispatch('user/setEcomStore', { 'productStoreId': event.detail.value })
           emitter.emit("productStoreChanged")
         }
       },
-      isSelected(select:any){
-        return this.$route.path===select
-      },
     },
     setup() {
       const store = useStore();
       const router = useRouter();
+      const appPages = [
+        {
+          title: "Create Rule",
+          url: "/select-product",
+          childRoutes: ['/select-product'],
+          iosIcon: optionsOutline,
+          mdIcon: optionsOutline
+        },
+        {
+          title: "Rule Pipeline",
+          url: "/threshold-updates",
+          childRoutes: ['/threshold-updates'],
+          iosIcon: pulseOutline,
+          mdicon: pulseOutline
+        },
+        {
+          title: "Settings",
+          url: "/Settings",
+          iosIcon: settingsOutline,
+          mdIcon: settingsOutline
```

Comment:
Add support for permission check, as we have support to check for permission for the pages